<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrepareExtractorDialog</class>
 <widget class="QDialog" name="PrepareExtractorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подготовка к экстракции</string>
  </property>
  <widget class="QTableView" name="measuresTableView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>14</y>
     <width>571</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="analysisTreeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
   <property name="indentation">
    <number>10</number>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Используемый метод:</string>
   </property>
  </widget>
  <widget class="QLabel" name="methodText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Method</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>14</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Библиотека:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="libraryComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>106</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>420</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
